### ISSUE TITLE
doc issue in AUTOGRAD MECHANICS

### ISSUE THREAD

**Issue by bhbbbbb** [pytorch/pytorch#74682]
### 📚 The doc issue

In the section **Evaluation Mode** (`nn.Module.eval()`)

https://pytorch.org/docs/stable/notes/autograd.html#evaluation-mode-nn-module-eval

> Functionally, `module.eval()` (or equivalently `module.train()`) are completely orthogonal to no-grad mode and inference mode.

Should be `module.train(False)`

### Suggest a potential alternative/fix

Functionally, `module.eval()` (or equivalently `module.train(False)`) are completely orthogonal to no-grad mode and inference mode.

cc @brianjo @mruberry

**Comment by VitalyFedyunin** [comment]
Feel free to submit PR with docs update.
